packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -62,10 +63,19 @@ const BlockDraggableWrapper = ( { children } ) => {
 	const isDragging = useSharedValue( false );
 	const scrollAnimation = useSharedValue( 0 );
 
+	const [
+		startScrolling,
+		scrollOnDragOver,
+		stopScrolling,
+		draggingScrollHandler,
+	] = useScrollWhenDragging();
+
 	const scrollHandler = ( event ) => {
 		'worklet';
 		const { contentOffset } = event;
 		scroll.offsetY.value = contentOffset.y;
+
+		draggingScrollHandler( event );
```

Comment:
`useScrollWhenDragging` hook also needs to listen to scroll events, hence we connect its scroll handler with the one defined in `BlockDraggable`.